packaging/systemd/package.sh

```diff
@@ -2,27 +2,23 @@
 
 get_version() {
     pushd ../../systemd
-    version=1.0.0~$(git describe --always --tags --dirty --match "[0-9]*.[0-9]*.[0-9]*")
+    version=1.0.0-$(git log --date=format:%Y%m%d --pretty=dirty~git%cd.%h -n 1)
```

Comment:
Does this cause any issue if the first character after the dash is not an incrementing number indicating debian_revision?

Reply:
The debian package is built. I have not tested yet if it is possible to be installed.

Reply:
If it is a problem I will replace it with the following format:
version=1.0.0-**0**$(git log --date=format:%Y%m%d --pretty=~git%cd.%h**-dirty** -n 1)

Reply:
Updated.